lib/Installation/Partitioner/LibstorageNG/v4_3/ConfirmationWarning.pm

```diff
@@ -34,4 +34,22 @@ sub press_yes {
     return $self->{btn_yes}->click();
 }
 
+=head2 is_shown
```

Comment:
Could you delete all references to is_shown, so we can avoid someone else to use it in the wrong way plus it is dead code that no scenario would be using it.

Reply:
Changed.